src/routes/adminPanel/adminPanelTable/tablePagination.component.tsx

```diff
@@ -0,0 +1,72 @@
+interface TablePaginationProps {
+  gotoPage: (page: number) => void;
+  previousPage: () => void;
+  nextPage: () => void;
+  canPreviousPage: boolean;
+  canNextPage: boolean;
+  pageCount: number;
+  pageIndex: number;
+  pageOptions: number[];
+  pageSize: number;
+  setPageSize: (size: number) => void;
+}
+
+export const TablePagination = ({
+  gotoPage,
+  previousPage,
+  nextPage,
+  canPreviousPage,
+  canNextPage,
+  pageCount,
+  pageIndex,
+  pageOptions,
+  pageSize,
+  setPageSize,
+}: TablePaginationProps) => {
+  return (
+    <div className="pagination">
+      <button onClick={() => gotoPage(0)} disabled={!canPreviousPage}>
+        {'<<'}
+      </button>{' '}
+      <button onClick={() => previousPage()} disabled={!canPreviousPage}>
```

Comment:
you can just pass onClick={previousPage} without recreating this fn every render